census/app.js

```diff
@@ -116,10 +117,13 @@ function start() {
 
   routes.utils.setupAuth();
 
-  app.get('models').umzug.up().then(function() {
-    app.listen(app.get('port'), function() {
-      console.log("Listening on " + app.get('port'));
-    });
+  return new Promise(function(resolve, reject) {
```

Comment:
Because in a tests I need to know when server have started - it may take undefined amount of time, that's why I decided to use promise. See https://github.com/kravets-levko/opendatacensus/blob/feature/updates-to-api/tests/api.js#L14